packages/terra-functional-testing/src/services/wdio-terra-service/GithubIssue.js

```diff
@@ -0,0 +1,28 @@
+const { Octokit } = require('@octokit/core');
+
+class GithubIssue {
+  constructor(url, token, owner, repo, number) {
+    this.ok = new Octokit({ baseUrl: url, auth: token });
```

Comment:
Can we rename `this.ok` to `this.octoKit` or something similar so it's more legible?

Reply:
Sure no problem

Reply:
done